includes/Modules/AdSense/AMP_Tag.php

```diff
@@ -66,6 +66,39 @@ public function register() {
 		$this->do_init_tag_action();
 	}
 
+	/**
+	 * Gets the attributes for amp-story-auto-ads and amp-auto-ads tags.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $type Whether it's for web stories. Can be `web-story` or ``.
+	 * @return array Filtered $options.
+	 */
+	private function get_auto_ads_attributes( $type = '' ) {
+		$options = array(
+			'ad-client' => $this->tag_id,
+		);
+
+		if ( 'web-story' === $type && ! empty( $this->story_ad_slot_id ) ) {
+			$options['ad-slot'] = $this->story_ad_slot_id;
+		}
+
+		$hook_name        = 'web-story' === $type ? 'googlesitekit_amp_story_auto_ads_attributes' : 'googlesitekit_amp_auto_ads_attributes';
+		$filtered_options = apply_filters(
+			$hook_name,
+			$options,
+			$this->tag_id,
+			$this->story_ad_slot_id
+		);
```

Comment:
@asvinb let's better separate filters because it will be easier to read it and understand what happens. Plus, it will be consistent with all other filters that we use in our codebase.

Reply:
ah makes sense. Over to you for a last time 🤞